docs/reference/modules/scripting/security.asciidoc

```diff
@@ -132,124 +125,9 @@ script.engine.painless.inline:          true <2>
 script.engine.painless.stored.search:   true <3>
 script.engine.painless.stored.aggs:     true <3>
 
-script.engine.mustache.stored.search: true <4>
+script.engine.mustache.stored.search:   true <4>
 -----------------------------------
 <1> Disable all scripting from any source.
-<2> Allow inline Groovy scripts for all operations
-<3> Allow stored Groovy scripts to be used for search and aggregations.
+<2> Allow inline Painless scripts for all operations
+<3> Allow stored Painless scripts to be used for search and aggregations.
 <4> Allow stored Mustache templates to be used for search.
-
-[[java-security-manager]]
-[float]
-=== Java Security Manager
-
```

Comment:
After another read it look like nothing can be merged. All the tips here are basically ways to expose more classes to groovy. The words in plugin/authors.asciidoc are more about making the security manager work for your plugin.

I wonder if it makes sense to only target this at 6.0? Or just cherry-pick *some* of it back to 5.x? The page is pretty out of date for 5.x but all this security manager stuff still kind of applies to groovy.